api/rpc/service/service.proto

```diff
@@ -35,3 +18,25 @@ message PublishSpec {
   uint32 publish_port = 3;
   uint32 internal_port = 4;
 }
+
+message ServiceCreateRequest {
+  ServiceSpec service_spec = 1;
+}
+
+message ServiceCreateResponse {
+  string id = 1;
+}
+
+message RemoveRequest {
+  string ident = 1;
```

Comment:
making it clear it can be an ID, short ID, or name. Actually, that was something that @freignat91 recommended. I have no problem calling it `id`, which matches docker docs.